eng/Versions.props

```diff
@@ -1,27 +1,27 @@
 <Project>
   <PropertyGroup>
     <!-- Current previous .NET SDK major version's stable release of MAUI packages -->
-    <MicrosoftMauiPreviousDotNetReleasedVersion>6.0.419</MicrosoftMauiPreviousDotNetReleasedVersion>
+    <MicrosoftMauiPreviousDotNetReleasedVersion>6.0.511</MicrosoftMauiPreviousDotNetReleasedVersion>
```

Comment:
I updated this to get this fix in `net6.0` MAUI apps: 3d9a27c0b206495130aa8a813b1aae5a36b54ca5